flatpak.yaml

```diff
@@ -0,0 +1,70 @@
+##########
+# Freeciv21 - flatpak.yaml
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: James Robertson <jwrober@gmail.com>
+#
+# Build Steps:
+#   - (once) sudo apt install flatpak-builder
+#   - (once) sudo flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+#   - flatpak-builder --install-deps-from=flathub --force-clean build/flatpak dist/flatpak.yaml
+#   - flatpak-builder --user --install --force-clean build/flatpak dist/flatpak.yaml
+#   - flatpak run net.longturn.freeciv21
+#   - flatpak remove net.longturn.freeciv21
+#
+##########
+
+app-id: net.longturn.freeciv21
+runtime: org.kde.Platform
+runtime-version: '5.15'
```

Comment:
this runtime is EOL

Reply:
Thanks so much for the quick review. I'll have a look at these. Is it OK to push more commits to the branch?

Reply:
Updated to same runtime as the KDE apps use.